systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaRebalanceUtils.java

```diff
@@ -81,7 +80,7 @@ public static void doRebalancingProcess(String namespaceName, String rebalanceNa
 
     public static void doRebalancingProcess(String namespaceName, String rebalanceName, boolean autoApproval) {
         LOGGER.info(String.join("", Collections.nCopies(76, "=")));
-        LOGGER.info(KafkaRebalanceResource.kafkaRebalanceClient().inNamespace(namespaceName).withName(rebalanceName).get().getStatus().getConditions().get(0).getType());
+        LOGGER.info(kafkaRebalanceClient().inNamespace(namespaceName).withName(rebalanceName).get().getStatus().getConditions().get(0).getType());
         LOGGER.info(String.join("", Collections.nCopies(76, "=")));
```

Comment:
Could you please improve the log message a little bit? 

```
===========================================
ProposalReady
===========================================
```

is kinda useless from my POV. Same few lines bellow.

Reply:
why do I need to change it here when it's not even related to the PR? I would focus on the removal of RM instead of all things that are in the code for many years and nobody touched that since 2020

Reply:
Because you are changing the line anyway

Reply:
It's just path to the client changed in that line. If you want to have it changed, I will have a look at it in different PR rather than doing all not-related things in one.

Reply:
Ok